pytket/docs/requirements.txt

```diff
@@ -1,4 +1,5 @@
 sphinx >= 4.5
 sphinx_autodoc_annotation >= 1.0
-sphinx_book_theme >= 0.3
+sphinx_book_theme == 0.3.3
```

Comment:
```suggestion
sphinx_book_theme ~= 0.3.3
```

Reply:
Done, will upgrade to the new release of the docs theme in a separate PR.